default/hypr/bindings/utilities.conf

```diff
@@ -1,5 +1,5 @@
 # Launching
-bind = SUPER, space, exec, pkill wofi || wofi --show drun --sort-order=alphabetical --style="$HOME/.local/share/omarchy/default/wofi/search.css"
+bind = SUPER, space, exec, pkill wofi || wofi --show drun --sort-order=alphabetical --style="$HOME/.config/wofi/style.css"
```

Comment:
I think this needs to stay as it is. And then we load the user-side css from that file.